### PR TITLE
Task runner backend system.

### PR DESCRIPTION
**Pull request by jproffitt**

### CODE REVIEW

File: fabric_bolt/task_runners/__init__.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from django.conf import settings
+from django.utils.module_loading import import_string
+
+__author__ = 'jproffitt'
```
Comment (npardington) [inline fabric_bolt/task_runners/__init__.py]: Probably auto generated cruft

Comment (jproffitt) [inline fabric_bolt/task_runners/__init__.py]: Oops. I'll remove it.

File: requirements/base.txt [hunk 2]
```diff
@@ -2,7 +2,8 @@ django==1.8.4
 pillow==2.9.0
 django-stronghold==0.2.7
 django-crispy-forms==1.5.1
-django-authtools==1.2.0
+#django-authtools==1.2.1
+git+https://github.com/fusionbox/django-authtools.git@8f040b9644d0cd2e7694624d9e6f93a0fc5baf0d
```
Comment (npardington) [inline requirements/base.txt]: why?

Comment (jproffitt) [inline requirements/base.txt]: Authtools was using a deprecated api which didn't work with channels. I don't know when 1.2.1 will be released. the maintainer seems to be silent.

**Pull request merged**